docs/setup/security/notebook_authorization.md

```diff
@@ -81,3 +81,11 @@ For example, before executing a **Read** operation, it checks if the user and th
 Zeppelin executes a [REST API call](https://github.com/apache/zeppelin/blob/master/zeppelin-server/src/main/java/org/apache/zeppelin/rest/NotebookRestApi.java) for the notebook permission information.
 In the backend side, Zeppelin gets the user information for the connection and allows the operation if the users and groups
 associated with the current user have at least one entity that belongs to owner entities for the notebook.
+
+## Permissions for folders
+Folder with permissions is way to unite notes with the same permissions and change the permissions together.
+
+Folder without permissions (with empty permissions) continues to work like common folder.
+
+Folder with permissions sets permissions to all its content (subfolders, notes). Nobody can change permissions on content of folder with permissions. Permissions are regulated by folder.
+Only owners of the folder can change its permissions. If permissions is empty (folder without permissions), anybody can set permissions. Owners of the folder can remove all permissions, then folder becomes 'folder without permissions'.
```

Comment:
It's quite tricky to understand the total view. Could you write a table for the simplified view?

Reply:
and What happens if 

- `folder1/note1` has owner
- but `folder1` doesn't have permission
- then anyone can set permission?